### PR TITLE
net-ftp/lftp: fix QA warning

### PR DESCRIPTION
**Pull request by ConiKost**
Closes: https://bugs.gentoo.org/665034
Signed-off-by: Conrad Kostecki <conrad@kostecki.com>
Package-Manager: Portage-2.3.50, Repoman-2.3.11

### CODE REVIEW

**Comment by a17r** [comment]
This PR needs rebase.

@jer-gentoo ping

**Comment by ConiKost** [comment]
> This PR needs rebase.

Done.

File: net-ftp/lftp/lftp-9999.ebuild [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
-inherit autotools eutils git-r3 libtool
+inherit autotools eutils git-r3 gnome2-utils libtool
```
Comment (a17r) [inline net-ftp/lftp/lftp-9999.ebuild]: these days this is done by `xdg-utils` instead of `gnome2-utils`

File: net-ftp/lftp/lftp-9999.ebuild [hunk 2]
```diff
@@ -95,3 +95,11 @@ src_install() {
 		use ${script} || { rm "${ED}"/usr/share/${PN}/${script} || die ;}
 	done
 }
+
+pkg_postinst() {
+	gnome2_icon_cache_update
```
Comment (a17r) [inline net-ftp/lftp/lftp-9999.ebuild]: `xdg_icon_cache_update`

File: net-ftp/lftp/lftp-9999.ebuild [hunk 3]
```diff
@@ -95,3 +95,11 @@ src_install() {
...
+}
+
+pkg_postrm() {
+	gnome2_icon_cache_update
```
Comment (a17r) [inline net-ftp/lftp/lftp-9999.ebuild]: `xdg_icon_cache_update`

File: net-ftp/lftp/lftp-9999.ebuild (hunk 1 above)
Comment (ConiKost) [reply net-ftp/lftp/lftp-9999.ebuild]: Thanks, Done.

File: net-ftp/lftp/lftp-9999.ebuild (hunk 2 above)
Comment (ConiKost) [reply net-ftp/lftp/lftp-9999.ebuild]: Thanks, Done.

File: net-ftp/lftp/lftp-9999.ebuild (hunk 3 above)
Comment (ConiKost) [reply net-ftp/lftp/lftp-9999.ebuild]: Thanks, Done.

**Comment by a17r** [comment]
Looks like he was inspired to do that now... https://github.com/gentoo/gentoo/commit/97e3267240185760244506ad14b13d6f9e3ccc2f

**Pull request closed without merging**